boards/shields/mikroe_eth_click/mikroe_eth_click.overlay

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) 2020 Linaro Limited.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&mikrobus_spi {
+	status = "okay";
+	cs-gpios = <&mikrobus_header 2 0>; /* CS */
+
+	eth_click@0 {
+		compatible = "microchip,enc28j60";
+		local-mac-address = [00 00 00 01 02 03];
+		/* Errata B7/1 specifies min 8Mhz, 20MHz max according to RM */
```

Comment:
I wasn't aware of this issue. It might be worth adding to the shield documentation to highlight it there?

Page 3: http://ww1.microchip.com/downloads/en/DeviceDoc/80349c.pdf
```
When the SPI clock from the host microcontrolleris run at frequencies of
less than 8 MHz, reading orwriting to the MAC registers may be unreliable.
```

Reply:
@microbuilder there will be multiple erratas that were observed in a future pull on enc28j60 driver, not sure if documenting them in driver/shield's page will be as useful as commenting them where a code patch is applied.
In this case, where speed is set, we have min/max according to errata/manual.

Do you mean adding errata's headline in the comment, rather than just a reference?

Reply:
It's still useful here, I agree, and we can't comment every errata, I just didn't realize this one existed and I've used this chip quite a bit in the past. Glad you commented it here, though.